serde/core.py

```diff
@@ -502,11 +502,11 @@ def supports_default(self) -> bool:
 F = TypeVar('F', bound=Field)
 
 
-def fields(field_cls: Type[F], cls: Type) -> Iterator[F]:
+def fields(field_cls: Type[F], cls: Type) -> List[F]:
```

Comment:
Does this have to be a list for the templating to work?

Reply:
Yeah, it was used twice in the template. Returning `List` is more convenient.  Thanks for reviewing!